src/FormMigration/Controllers/TransferController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Give\FormMigration\Controllers;
+
+use Give\DonationForms\V2\Models\DonationForm;
+use Give\FormMigration\Actions\TransferDonations;
+use Give\FormMigration\Actions\TransferFormUrl;
+use Give\FormMigration\DataTransferObjects\TransferOptions;
+use Give\Framework\Database\DB;
+use Give\Framework\QueryBuilder\QueryBuilder;
+use WP_REST_Request;
+use WP_REST_Response;
+
+class TransferController
+{
+    protected $debugContext;
+
+    /**
+     * @var WP_REST_Request
+     */
+    protected $request;
+
+    public function __construct(WP_REST_Request $request)
+    {
+        $this->request = $request;
+    }
+
+    public function __invoke(DonationForm $formV2, TransferOptions $options)
+    {
+        DB::transaction(function() use ($formV2, $options) {
+            global $wpdb;
+            $v3FormId = DB::get_var(
+                DB::prepare(
+                    "
+                    SELECT `form_id`
```

Comment:
Not a big deal, but since you already use the DB class, you might also use QueryBuilder. But this is also fine.